tools/interactive/interactivetool_odv.xml

```diff
@@ -0,0 +1,38 @@
+<tool id="interactive_tool_odv" tool_type="interactive" name="ODV" version="@VERSION@">
+    <description>interactive plotting tool for geo-referenced data</description>
+    <macros>
+    <token name="@VERSION@">5.6.5</token>
+    </macros>
+    <requirements>
+        <container type="docker">quay.io/mariejo/docker_odv</container>
```

Comment:
Please note that the containers should be versioned ideally.

Reply:
Would be nice you can do `docker tag quay.io/mariejo/docker_odv quay.io/mariejo/docker_odv:5.6.5` and than `docker push quay.io/mariejo/docker_odv quay.io/mariejo/docker_odv:5.6.5`